tests/pages/firefox/do_not_track.py

```diff
@@ -31,15 +31,23 @@ class FrequentlyAskedQuestion(PageRegion):
 
     _question_locator = (By.CSS_SELECTOR, 'h3[role="tab"]')
     _answer_locator = (By.CSS_SELECTOR, 'div[role="tabpanel"]')
+    _expanded_locator = (By.CSS_SELECTOR, 'h3[aria-expanded="true"]')
 
     def show_answer(self):
         assert not self.is_answer_displayed, 'Answer is already displayed'
-        el = self.root.find_element(*self._answer_locator)
-        self.root.find_element(*self._question_locator).click()
-        Wait(self.selenium, self.timeout).until(expected.visibility_of(el))
+        question = self.root.find_element(*self._question_locator)
+        answer = self.root.find_element(*self._answer_locator)
+        # Prevent Selenium automatically scrolling this element to the top of
+        # the page, which can lead to unintented clicks on the sticky header
+        self.selenium.execute_script('arguments[0].scrollIntoView(false);', question)
+        question.click()
+        Wait(self.selenium, self.timeout).until(expected.visibility_of(answer))
 
     def hide_answer(self):
         assert self.is_answer_displayed, 'Answer is already hidden'
+        # Wait until the accordion is fully expanded before clicking it again
+        Wait(self.selenium, self.timeout).until(
+            expected.presence_of_element_located(self._expanded_locator))
```

Comment:
Rather than having another locator, we should reuse the answer locator and wait for it to have the appropriate attribute. Also, we should do this in `show_answer` after clicking the question, to prevent any further action until after the answer is fully displayed.

```python
answer = self.root.find_element(*self._answer_locator)
Wait(self.selenium, self.timeout).until(
    lambda m: answer.get_attribute('aria-expanded') == 'true')
```